examples/camera-app/linux/include/clusters/webrtc_provider/webrtc-provider-manager.h

```diff
@@ -18,10 +18,15 @@
 
 #pragma once
 
+#include "camera-device-interface.h"
 #include <app-common/zap-generated/cluster-enums.h>
 #include <app/CASESessionManager.h>
 #include <app/clusters/webrtc-transport-provider-server/webrtc-transport-provider-server.h>
+#include <media-controller.h>
 #include <rtc/rtc.hpp>
+#include <transport/webrtc-transport.h>
```

Comment:
remove transport directory to align its cpp, it looks webrtc-transport is only used within webrtc-provider-manager, if so, we should put it under clusters/webrtc_provider